tests/integration-tests/tests/schedulers/test_awsbatch.py

```diff
@@ -18,6 +18,7 @@
 from tests.common.schedulers_common import AWSBatchCommands
 
 
+@pytest.mark.batch_dockerfile_deps
```

Comment:
An awsbatch mark is already added to this function by the testing framework (default markers are based on the filename). I'm ok with keeping this here so that if additional functions are added this is the only one being invoked for this kind of tests

Reply:
Doesn't using the awsbatch mark result in running all of the tests that support that scheduler?

Reply:
nope..a default marker is added based on the name of the file defining the test functions. E.g. test_awsbatch.py -> all functions have a default `@pytest.mark.awsbatch` marker

Reply:
Okay. I suppose I'll keep it in case we need to add tests for this in different files in the future.